jenkins-integration/Gemfile

```diff
@@ -1,3 +1,4 @@
 source ENV['GEM_SOURCE'] || 'https://rubygems.org'
 
-gem 'beaker', :github => 'puppetlabs/beaker', :branch => 'master'
+gem 'beaker', '2.15.1'
+gem 'scooter', :git => 'git@github.com:puppetlabs/scooter.git', :tag => '2.2'
```

Comment:
:clap: :clap: :clap: 

Turns out we don't *need* to use the internal rubygems mirror and deal with Gemfile conditionals and remembering to set GEM_SOURCE when calling Beaker scripts.  

Bundler can pull gems out of private GitHub repositories just fine. The only thing we lose is the ability to do version ranges, although you can point it to a branch. Either way we don't need version ranges for these gems, we can just pin to a release.